### PR TITLE
magento/devdocs#: Extend “Customize the 503 error page” topic name

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) has appeared from the question in cloud channel:

and adds a "Maintenance" world into "Customize the 503 error page" section of the [Customize error pages](https://devdocs.magento.com/cloud/cdn/cloud-fastly-custom-response.html) page.

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/cdn/cloud-fastly-custom-response.html

## Links to Magento source code

-  ["Set Error / Maintenance page guide" by Fastly](https://github.com/fastly/fastly-magento2/blob/master/Documentation/Guides/ERROR-MAINTENANCE-PAGE.md)

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/cdn/cloud-fastly-custom-response.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The HTML code for the custom page must meet the following requirements:
 -  Specify all CSS inline in the HTML source.
 -  Bundle images in the HTML page using base64 so that they display even if Fastly is offline. See [Data URIs on the css-tricks site](https://css-tricks.com/data-uris/).
 
-## Customize the 503 error page
```
Comment (rogyar) [inline src/cloud/cdn/cloud-fastly-custom-response.md]: There's at least one link in this document that goes to `(#customize-the-503-error-page)` (line 23). We need to make sure that we don't introduce any broken links with this change. 

Thank you

Comment (atwixfirster) [reply src/cloud/cdn/cloud-fastly-custom-response.md]: Golden eye, @rogyar !

**fixed** and **verified** via local preview.

Thank you!

**Comment by meker12** [comment]
@atwixfirster  Problem is that the page provides instructions for two types of error pages -- the WAF error page for 403 errors, and the 503 error pages.  So changing the title doesn't work.  However, we can probably put a more specific header in the page itself so that the content would show up in the search.

**Comment by atwixfirster** [comment]
> @atwixfirster Problem is that the page provides instructions for two types of error pages -- the WAF error page for 403 errors, and the 503 error pages. So changing the title doesn't work. However, we can probably put a more specific header in the page itself so that the content would show up in the search.

Hi, @meker12!

Tutorial in the `Customize the 503 error page` contains the same instruction like this one - ["Set Error / Maintenance page guide" by Fastly](https://github.com/fastly/fastly-magento2/blob/master/Documentation/Guides/ERROR-MAINTENANCE-PAGE.md).

`Customize the WAF error page` section describes how to edit WAF page only.

**Comment by meker12** [comment]
I think we should leave the content of the page as is, with no changes to the headers because a 503 error page can be for a timeout or maintenance as indicated in the bulleted explanation.

A simpler solution is to change the page title to "Customize error and maintenance pages".

**Comment by meker12** [comment]
> > @atwixfirster Problem is that the page provides instructions for two types of error pages -- the WAF error page for 403 errors, and the 503 error pages. So changing the title doesn't work. However, we can probably put a more specific header in the page itself so that the content would show up in the search.
> 
> Hi, @meker12!
> 
> Tutorial in the `Customize the 503 error page` contains the same instruction like this one - ["Set Error / Maintenance page guide" by Fastly](https://github.com/fastly/fastly-magento2/blob/master/Documentation/Guides/ERROR-MAINTENANCE-PAGE.md).
> 
> `Customize the WAF error page` section describes how to edit WAF page only.

See my alternative suggestion below.

**Comment by atwixfirster** [comment]
>A simpler solution is to change the page title to "Customize error and maintenance pages".

@meker12 , I've implemented your proposition. Could you please review?

Many thanks!

**Comment by meker12** [comment]
running tests

**Pull request merged**